app/javascript/App/bundles/NewsfeedPage/NewsList.tsx

```diff
@@ -58,30 +58,20 @@ class NewsList extends React.Component<Props, State> {
   }
 
   closeTips() {
-    this.props.newsfeedTips()
+    this.props.toggleNewsfeedTips()
```

Comment:
These two names are misleading: toggle or close?

Reply:
It was worked as `toggle` but used as `close`. I've changed the initial function and renamed all to `closeTips`